enterprise/signals.py

```diff
@@ -349,16 +351,23 @@ def enterprise_unenrollment_receiver(sender, **kwargs):     # pylint: disable=un
     """
     enrollment = kwargs.get('enrollment')
     enterprise_enrollment = models.EnterpriseCourseEnrollment.objects.filter(
-        course_id=enrollment.course.course_key,
-        enterprise_customer_user__user_id=enrollment.user.id,
+        course_id=enrollment.course_id,
+        enterprise_customer_user__user_id=enrollment.enterprise_customer_user.user_id,
     ).first()
     if enterprise_enrollment:
         logger.info(
-            f"Marking EnterpriseCourseEnrollment as unenrolled for user {enrollment.user} and "
-            f"course {enrollment.course.course_key}"
+            f"Marking EnterpriseCourseEnrollment as unenrolled for user "
+            f"{enrollment.enterprise_customer_user.user_id} and course {enrollment.course_id}"
         )
         enterprise_enrollment.unenrolled_at = localized_utcnow()
         enterprise_enrollment.save()
+        get_producer().send(
+            signal=COURSE_UNENROLLMENT_COMPLETED,
```

Comment:
Should we check if `COUSE_ENROLLMENT_COMPLETED` is not None before using it? It looks like it can _potentially_ be None depending on if there was an `ImportError`.

Reply:
The reason I didn't put the check on it was later on in the code, we have a check that doesn't connect the receiver unless it is not None. I can put another check in if this is not sufficient though, but this seems to do it already. 

`if COURSE_UNENROLLMENT_COMPLETED is not None:
    COURSE_UNENROLLMENT_COMPLETED.connect(enterprise_unenrollment_receiver)`